.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,15 @@
+name: Docker Hub build
+on:
+  push:
+    branches:
+      - "develop"
+      - "master"
+      - "release/*"
+      - "release/lavida"

```

Comment:
I need to remove this.

Reply:
yeah let's also not forget to add the trigger for carma-system-*